### ISSUE TITLE
mconfig fails to find libseccomp files on opensuse leap

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#1953]
## Version of Singularity:

master branch

### Expected behavior

 checking: libseccomp+headers... yes

### Actual behavior

 checking: libseccomp+headers... no

### 

I've installed several libseccomp packages, but mconfig does not seem to find the necessary files. Perhaps the files are somewhere mconfig is not looking?

$ zypper search libseccomp
Loading repository data...
Reading installed packages...

i+ | libseccomp-devel  | Development files for libseccomp, an enhanced Seccomp (mode 2) helper library
i+ | libseccomp-tools  | Utilities for the seccomp API
i+ | libseccomp2       | An enhanced Seccomp (mode 2) helper library
   | libseccomp2-32bit | An enhanced Seccomp (mode 2) helper library

**Comment by jscook2345** [comment]
On opensuse, the header is installed in a different location. Here's what `pkg-config` says for accessing it.

    $ pkg-config --libs --cflags libseccomp

    -I/usr/include/libseccomp -lseccomp

**Comment by jscook2345** [comment]
@mem @jmstover 

What do you think about the following change:

```
-   $tgtcc $user_cflags $ldflags -x c -o /dev/null - -lseccomp >/dev/null 2>&1; then
+   $tgtcc $user_cflags $ldflags -x c -o /dev/null - $(pkg-config --libs --cflags libseccomp) >/dev/null 2>&1; then
```

On centos:
```
[vagrant@localhost ~]$ pkg-config --libs --cflags libseccomp
 -lseccomp
```

On opensuse:
```
vagrant@vagrant-openSUSE-Leap:~/singularity> pkg-config --libs --cflags libseccomp
-I/usr/include/libseccomp -lseccomp
```

**Comment by dtrudg** [comment]
Closed via #4341
